python/eks/cluster/requirements.txt

```
@@ -0,0 +1,5 @@
+aws-cdk.aws-autoscaling
+aws-cdk.custom-resources
+aws-cdk.aws-ec2
+aws-cdk.aws-eks
+aws-cdk.aws-iam
```

```suggestion
aws-cdk-lib>=2.0.0
constructs>=10.0.0
```
Would you mind upgrading this to CDK v2 so that it's consistent with our other examples? I've added code suggestions with what should be all the changes you need to upgrade. Let me know if you have trouble though.